components/ranking-item-responsive.tsx

```
@@ -98,6 +98,118 @@ const RankingItemResponsive = memo(function RankingItemResponsive({ item, disabl
+    <>
+      <style dangerouslySetInnerHTML={{ __html: `
+        /* モバイルレイアウト用CSS（重要な部分のみ） */
+        @media (max-width: 640px) {
+          .ranking-item-responsive__content {
+            display: grid !important;
+            grid-template-columns: 120px 1fr auto !important;
+            grid-template-areas: 
+              "left-column details buttons"
+              "left-column details buttons"
+              "left-column stats stats" !important;
+            grid-template-rows: auto auto auto !important;
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)